tests/gatsby-starter-default/src/templates/default.js

```diff
@@ -8,45 +8,46 @@ import Layout from '../components/Layout';
 import NewsItem from '../components/NewsItem';
 
 const componentFor = data => {
+  const nodes = query => (query ? query['edges'] : []).map(edge => edge.node);
   if (data) {
-    if (data.ploneCollection) {
+    if (data['ploneCollection']) {
       return (
         <Folder
-          data={data.ploneCollection}
-          images={data.allPloneImage}
-          files={data.allPloneFile}
+          data={data['ploneCollection']}
+          images={nodes(data['allPloneImage'])}
+          files={nodes(data['allPloneFile'])}
         />
       );
-    } else if (data.ploneDocument) {
+    } else if (data['ploneDocument']) {
       return (
         <Document
-          data={data.ploneDocument}
-          images={data.allPloneImage}
-          files={data.allPloneFile}
+          data={data['ploneDocument']}
+          images={nodes(data['allPloneImage'])}
+          files={nodes(data['allPloneFile'])}
         />
       );
-    } else if (data.ploneEvent) {
+    } else if (data['ploneEvent']) {
       return (
         <Event
-          data={data.ploneEvent}
-          images={data.allPloneImage}
-          files={data.allPloneFile}
+          data={data['ploneEvent']}
+          images={nodes(data['allPloneImage'])}
+          files={nodes(data['allPloneFile'])}
         />
       );
-    } else if (data.ploneFolder) {
+    } else if (data['ploneFolder']) {
       return (
         <Folder
-          data={data.ploneFolder}
-          images={data.allPloneImage}
-          files={data.allPloneFile}
+          data={data['ploneFolder']}
+          images={nodes(data['allPloneImage'])}
+          files={nodes(data['allPloneFile'])}
         />
       );
-    } else if (data.ploneNewsItem) {
+    } else if (data['ploneNewsItem']) {
```

Comment:
Any particular reason for switching from `dot` accessors to `bracket` accessor for object properties?

Reply:
@ajayns Not really. Silly thing, but PyCharm complained less with brackets.

I agree that the codebase is not currently consistent on this.

Reply:
@ajayns Would you know, is there any global good practice on selecting between these?

Reply:
The general convention is to use the `dot` notation, while `bracket` is only used in the case where there are special characters in the object property name.